Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporterEditor.cs

```diff
@@ -15,6 +15,25 @@ namespace UnityEngine.Experimental.Input.Editor
     [CustomEditor(typeof(InputActionImporter))]
     internal class InputActionImporterEditor : ScriptedImporterEditor
     {
+        // Show a PropertyField with a greyed-out default text if the field is empty and not being edited.
+        // This is meant to communicate the fact that filling these properties is optional and that Unity will
+        // use reasonable defaults if left empty.
+        private void PropertyFieldWithDefaultText(SerializedProperty prop, GUIContent label, string defaultText)
```

Comment:
Can we move this to SerializedPropertyHelpers or somewhere like that? Having utility methods hidden inside classes like this makes them hard to find.